.github/workflows/mirror-dev.yml

```diff
@@ -0,0 +1,34 @@
+name: Mirror dev to GitLab
+
+on:
+  push:
+    branches:
+      - dev
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  mirror:
+    if: github.repository == 'accessibility-exchange/platform'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Git
+        run: |
+          git config --global user.name "${{ vars.GIT_USER }}"
+          git config --global user.email "${{ vars.GIT_EMAIL }}"
+          git remote add gitlab ${{ vars.GITLAB_URL }}
+
+      - name: Push to GitLab
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.GITLAB_SSH_KEY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
+          git push gitlab HEAD:development
```

Comment:
I'm not sure this is actually pushing tags. Following from this stack overflow post ( https://stackoverflow.com/a/3745250 ). I think what you'd like to do here is:

```bash
git push --atomic gitlab dev:development --follow-tags
```

Reply:
> Are we sure we want to mirror tags?
> 
> I don't see them included as triggers for deployment.

It just depends if you want a copy of the tags in GitLab or not.

Reply:
I can leave the workflow. It just has no impact as far as GitLab CI/CD automations at this point.